src/helpers/generateGuessQueue.ts

```diff
@@ -0,0 +1,30 @@
+import { gameUserController } from '../controllers';
+import { IRoom } from '../models';
+
+export const generateGuessQueue = async (room: IRoom) => {
+  for await (const participant of room.participants) {
+    const gameUserIndex = room.participants.findIndex(
+      (findParticipant) => findParticipant.id === participant.id
+    );
+
+    const gameUser = await gameUserController.getGameUser(null, {
+      id: participant._id,
+    });
+
+    if (gameUserIndex === 0) {
+      gameUser.namingUser = room.participants[room.participants.length - 1];
+      gameUser.seterUser = room.participants[gameUserIndex + 1];
```

Comment:
Pls extract `gameUserIndex + 1` and `gameUserIndex - 1` to variables
That will reduce code duplication

Reply:
And you could extract `room.participants` as well (to `participants` for example)